src/components/pages/ParentBooking/ParentBookingCard.js

```diff
@@ -17,7 +20,28 @@ const ParentBookingCard = props => {
     location,
     instructor_name,
     size,
+    course_id,
+    child_id,
   } = props.booking;
+
+  const { authState } = useOktaAuth();
+  console.log(props);
+
+  const handleClick = e => {
+    axios
+      .post(
+        `http://localhost:8080/children/${child_id}/enrollments`,
+        course_id,
+        {
+          header: {
+            idToken: authState.idToken,
+          },
+        }
+      )
+      .then(res => console.log(res))
```

Comment:
You're not doing anything here, what do you want to happen after a child enrollment gets posted? Maybe a redirect?